src/components/pill-selector-dropdown/PillSelector.js

```diff
@@ -205,7 +205,7 @@ class PillSelector extends React.Component<Props, State> {
         };
 
         return (
-            <Tooltip isShown={hasError} text={error || ''} position="middle-right" theme="error">
+            <Tooltip isShown={hasError} text={error || ''} position="bottom-right" theme="error">
```

Comment:
Is this confirmed with design that all PillSelectors will be changed to bottom-right?

Reply:
we can add a `errorPosition` / `errorTooltipPosition` prop if needed

Reply:
Yes. Just in case, I went back and reviewed our conversation. The one difference is a slight preference for using `bottom-left` instead of `bottom-right`. I've gone ahead and made the change.